src/benchmarks/micro/libraries/System.Diagnostics.Tracing/Perf_EventListener.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+using System.Runtime.CompilerServices;
+
+namespace System.Diagnostics.Tracing
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class Perf_EventListener
+    {
+        private enum TestEnum
+        {
+            Foo = 123
+        }
+
+        private sealed class BenchmarkEventSource : EventSource
+        {
+            public BenchmarkEventSource() : base(nameof(Perf_EventListener)) { }
+
+            [Event(1)]
+            public void EventNoParams() => WriteEvent(1);
+
+            [Event(2)]
+            public void EventIntParams(int arg1, int arg2, int arg3) => WriteEvent(2, arg1, arg2, arg3);
+
+            [Event(3)]
+            public void EventStringParams(string arg1, string arg2, string arg3) => WriteEvent(3, arg1, arg2, arg3);
+
+            [Event(4)]
+#if NET
```

Comment:
what is `#if NET` ?

Reply:
I think it's essentially `NET5_0_OR_GREATER`